it/logs/test_logs.py

```diff
@@ -22,6 +22,7 @@
 pytest_rally = pytest.importorskip("pytest_rally")
 
 
+@pytest.mark.track("elastic/logs")
```

Comment:
Shouldn't this be `elastic` as we're only looking at top-level directories?

Reply:
For the record I changed my mind. `@pytest.mark.track("elastic/logs")` makes sense to me now because `elastic` is not a track name, while `elastic/logs` is. Sorry for the confusion. Only `--track-filter=elastic/logs` filtering all of `elastic/*` does not make sense, as commented in https://github.com/elastic/pytest-rally/pull/7#discussion_r2388254074.